valhalla/jawn/src/controllers/public/heliconeSqlController.ts

```diff
@@ -27,6 +28,26 @@ export interface ExecuteSqlRequest {
   sql: string;
 }
 
+export interface CreateSavedQueryRequest {
+  name: string;
+  sql: string;
+}
+
+export interface UpdateSavedQueryRequest {
+  id: string;
+  name: string;
+  sql: string;
+}
```

Comment:
feels a bit repetitive - could go 
```suggestion
export interface CreateSavedQueryRequest {
  name: string;
  sql: string;
}

export interface UpdateSavedQueryRequest extends CreateSavedQueryRequest {
  id: string;
}
```

this is like extremely nitpicky so honestly just ignore but something to think about cause in this case if you ever change the `CreateSavedQueryRequest` you probably want to have the same attribute updated in the `UpdateSavedQueryRequest` as well uk?